e2e/pod.go

```diff
@@ -88,21 +88,43 @@ func waitForDeploymentComplete(name, ns string, c kubernetes.Interface, t int) e
 	return nil
 }
 
-func getCommandInPodOpts(f *framework.Framework, c, ns string, opt *metav1.ListOptions) (framework.ExecOptions, error) {
+func getCommandInPodOpts(f *framework.Framework, c, ns string, cn string, opt *metav1.ListOptions) (framework.ExecOptions, error) {
+	e2elog.Logf("In getCommandInPodOpts: %v", cn)
+	pn := ""
 	cmd := []string{"/bin/sh", "-c", c}
 	podList, err := f.PodClientNS(ns).List(context.TODO(), *opt)
 	framework.ExpectNoError(err)
 	if len(podList.Items) == 0 {
 		return framework.ExecOptions{}, errors.New("podlist is empty")
 	}
+	found := false
```

Comment:
find Container Name in the pod can be wrapped into a new function .

Reply:
Done now!

Reply:
Thanks @pkalever :+1: